program-test/tests/warp.rs

```diff
@@ -29,89 +30,13 @@ use {
         transaction::{Transaction, TransactionError},
     },
     solana_stake_program::stake_state,
-    solana_vote_program::{
-        vote_instruction,
-        vote_state::{self, VoteInit, VoteState},
-    },
+    solana_vote_program::vote_state::{self},
```

Comment:
micro-nit:
```suggestion
    solana_vote_program::vote_state,
```

Reply:
It looks like this nit didn't make it in

Reply:
updated.